src/main/java/org/opentripplanner/routing/core/vehicle_sharing/VehicleDescription.java

```diff
@@ -143,25 +143,17 @@ protected Double getMaximumRangeInMeters() {
     }
 
     @JsonIgnore
-    public VehiclePricingPackage getVehicleSharingPackage(int index) {
+    public VehiclePricingPackage getVehiclePricingPackage(int index) {
         return vehiclePricingPackages.get(index);
     }
 
-    public int getActivePackageIndex() {
```

Comment:
teraz się zastanawiam jakim cudem ostatnio tego nie zauważyłem - `VehicleDescription` musi być niemutowalny, bo jest współdzielony pomiędzy wszystkie zapytania. Trzeba usunąć z niego zmienną `activePackageIndex` i przenieść to na przykład do `State`.

Reply:
To drobiazg - po usunięciu tego atrybutu wystarczyło tylko zmodyfikować testy, bo tylko tam zostały jeszcze odwołania do niego, reszta jest w stanie. Zrobione :)